include/tester/db/Database.hpp

```diff
@@ -14,114 +15,35 @@
-
-    //     bsoncxx::builder::stream::array result_array{};
-
-    //     // for (const auto& document : documents) {
-    //     //     bsoncxx::document::view view = document.view();
-    //     //     std::stringstream header;
-    //     //     // header << "_id: " << view["_id"].get_string().value.to_string() << "\ncreation-time: " << view["creation-time"].get_string().value.to_string();
-    //     //     header << "_id: " << view["_id"].get_string().value.to_string();
-    //     //     results.push_back(header.str());
-    //     // }
-
-    //     for (const auto& doc_value : documents) {
-    //         bsoncxx::document::view doc = doc_value.view();
-
-    //         auto id_value = doc["_id"];
-    //         auto creation_time_value = doc["creation-time"];
-
-    //         // Check if both "_id" and "creation-time" fields are present in the document
-    //         if (!id_value || !creation_time_value) {
-    //             continue;
-    //         }
-
-    //         bsoncxx::builder::stream::document id_doc;
-    //         id_doc << "_id" << id_value.get_oid();
-    //         result_array << id_doc;
-
-    //         bsoncxx::builder::stream::document creation_time_doc;
-    //         creation_time_doc << "creation-time" << creation_time_value.get_date();
-    //         result_array << creation_time_doc;
-    //     }
-    //     // return results;
-    //     return bsoncxx::document::value{bsoncxx::builder::stream::open_array << result_array << bsoncxx::builder::stream::close_array << bsoncxx::builder::stream::finalize};
-    // }
-
-    // std::vector<bsoncxx::document::value> get_all_headers(const std::string& collection_name)
-    // {
-    //     mongocxx::collection collection = _database[collection_name];
-    //     mongocxx::cursor cursor = collection.find({});
-
-    //     std::vector<bsoncxx::document::value> results;
-    //     bsoncxx::builder::stream::array result{};
-    //     for (auto&& doc : cursor) {
-    //         result << bsoncxx::builder::stream::open_document;
-    //         if(doc["_id"])
-    //         {
-    //             // std::cout << doc["_id"].get_oid().value.to_string() <<std::endl;
-    //             result << "_id" << doc["_id"].get_oid().value.to_string();
-    //         }
-
-    //         if(doc["creation-time"])
-    //         {
-    //             // std::cout << doc["creation-time"].get_string().value.to_string() <<std::endl;
-    //             result << "creation-time" << doc["creation-time"].get_string().value.to_string();
-    //         }   
-            
-    //         result << bsoncxx::builder::stream::close_document;
-    //         // std::cout << bsoncxx::to_json(result.view());
-
-    //         // results.emplace_back(std::move(doc));
-
-    //     }
-
-    //     return result;
-    // }
-    bsoncxx::document::value getAllHeaders(const std::string& collection_name)
+    bsoncxx::document::value getAllHeaders(std::string collectionName)
```

Comment:
same here regarding the `static constexpr`

Reply:
done